content/en/docs/collector/install/binary/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Install from a Collector binary
```

Comment:
@tiffany76, let's avoid repeating "Install" in each subpage for the `linkTitle`. How about this:

```suggestion
title: Install from a Collector binary
linkTitle: From binary
```